conf/configuration.go

```diff
@@ -365,3 +371,13 @@ func (t *MessagebirdProviderConfiguration) Validate() error {
 	}
 	return nil
 }
+
+func (t *TextLocalProviderConfiguration) Validate() error {
+	if t.ApiKey == "" {
+		return errors.New("Missing Messagebird access key")
```

Comment:
this should be textlocal instead of messagebird?

Reply:
updated - btw I am new to golang, so not very confident about the test cases, I tried looking for twillio and messagebird, test cases but could not find any. can you share the same if possible.

Reply:
sure, we currently don't have an end-to-end test setup in gotrue for the sms providers but you can test it manually by doing the following steps:
1. set the correct env vars for enabling the textlocal provider
2. start gotrue locally 
3. hit the /otp endpoint and check that you receive an sms from textlocal 
4. hit the /verify endpoint with the otp received and check that the user is verified